src/components/Main/ContactUs.js

```diff
@@ -0,0 +1,126 @@
+import React,{useState} from 'react';
+import axios from 'axios';
+import decoration from '../../assets/Decoration.svg';
+import FacebookIcon from '../../assets/Facebook.svg'
+import InstagramIcon from '../../assets/Instagram.svg'
+
+const ContactUs=()=>{
+    
+    const [name, setName] = useState('');
+    const [mail, setMail] = useState('');
+    const [message, setMessage] = useState('');
+    const [errname, setErrname] = useState('');
+    const [errmail, setErrmail] = useState('');
+    const [errmsg, setErrmsg] = useState('');
+    const [subsuccess, setSubsuccess] = useState(``);
+    const [wrongname,setWrongName] = useState('');
+    const [wrongmail,setWrongMail] = useState('');
+    const [wrongmsg,setWrongMsg] = useState('');
+
+    const contact = {
```

Comment:
You can use shorthand for that, 

`const contact = {
name,
email,
message,
}
`

If keys are matching it will automatically do that